From 96abfc67c5384082c9f292be29c7ad088f0c63eb Mon Sep 17 00:00:00 2001
From: Guennadi Liakhovetski <g.liakhovetski@gmx.de>
Date: Thu, 3 Nov 2011 10:12:36 +0100
Subject: PM / QoS: Remove redundant check

Remove an "if" check, that repeats an equivalent one 6 lines above.

Signed-off-by: Guennadi Liakhovetski <g.liakhovetski@gmx.de>
Signed-off-by: Rafael J. Wysocki <rjw@sisk.pl>
(cherry picked from commit 6513fd6972f725291ee8ce62c7a39fb8a6c7391e)

Signed-off-by: Simon Horman <horms@verge.net.au>
---
 kernel/power/qos.c |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/kernel/power/qos.c b/kernel/power/qos.c
index 2c0a65e..56db751 100644
--- a/kernel/power/qos.c
+++ b/kernel/power/qos.c
@@ -387,8 +387,7 @@ static int pm_qos_power_open(struct inode *inode, struct file *filp)
 		pm_qos_add_request(req, pm_qos_class, PM_QOS_DEFAULT_VALUE);
 		filp->private_data = req;
 
-		if (filp->private_data)
-			return 0;
+		return 0;
 	}
 	return -EPERM;
 }
-- 
1.7.10.1.362.g242cab3

